weathervane.config.docker.micro.oneHost

```diff
@@ -0,0 +1,17 @@
+{
+  "dockerNamespace" : "yourRepository",
+  
+  "numDrivers": 8,
```

Comment:
micro.oneHost probably does not need 8 drivers?

Reply:
Good catch.  I fixed this.